components/authz-service/server/v2/project.go

```diff
@@ -595,13 +595,13 @@ func fromAPIType(t api.ProjectRuleTypes) (storage.RuleType, error) {
 func rulesToProjectRules(rules []engine.Rule) ([]*api.ProjectRule, error) {
 	conditions := make([]*api.Condition, len(rules))
 	for j, rule := range rules {
-		conditionType, exists := api.ProjectRuleConditionTypes_value[rule.Type]
+		conditionAttribute, exists := api.ProjectRuleConditionAttributes_value[rule.Type]
 		if !exists {
 			return []*api.ProjectRule{}, errors.New(fmt.Sprintf("Condition type %s is not supported", rule.Type))
```

Comment:
The previous comment reflects here as well: this text should say "condition attribute" not "condition type".

Reply:
This is getting deleted in https://github.com/chef/automate/pull/517